meshroom/nodes/aliceVision/ApplyCalibration.py

```diff
@@ -23,10 +23,17 @@ class ApplyCalibration(desc.AVCommandLineNode):
         desc.File(
             name="calibration",
             label="Calibration",
-            description="Calibration SfMData file.",
+            description="Calibration file (Either SfmData or Lens calibration file).",
             value="",
             uid=[0],
         ),
+        desc.BoolParam(
+            name="useJson",
+            label="Use lens calibration file",
+            description="Use Lens calibration file generated using 3Dequalizer.",
```

Comment:
```suggestion
            label="Use Lens Calibration File",
            description="Calibration is a Lens calibration file generated using 3Dequalizer instead of an sfmData.",
```